controllers/components/dashboard/dashboard_controller.go

```diff
@@ -99,6 +100,10 @@ func NewComponentReconciler(ctx context.Context, mgr ctrl.Manager) error {
 			updatestatus.WithSelectorLabel(labels.ComponentPartOf, componentsv1.DashboardInstanceName),
 		)).
 		WithAction(updateStatus).
+		// must be the final action
+		WithAction(gc.NewAction(
+			gc.WithLabel(labels.ComponentPartOf, componentsv1.DashboardInstanceName),
```

Comment:
should here be `gc.WithUnremovables(gvk.OdhDashboardConfig)`  ?

Reply:
done